app/models/report.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+# == Schema Information
+#
+# Table name: reports
+#
+#  id          :integer          not null, primary key
+#  reporter_id :integer
+#  reportee_id :integer
+#  reasons     :text
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#  comment_id  :integer
+#  user_id     :integer
```

Comment:
I have a suggestion for you here. I see that you are currently storing both the reporter and the reportee user ids, but also there's this extra user_id field. I believe this is not needed to store the "owner" of this report, because you can use the reporter_id for that purpose. In that case, you should also update the relationship in the model.